Mu2eG4/geom/geom_Wideband4modules_v3.txt

```diff
@@ -0,0 +1,536 @@
+
+int             crs.nModulesW1                   =   1;
+int             crs.nModulesW2                   =   1;
+int             crs.nModulesW3                   =   1;
+int             crs.nModulesW4                   =   1;
+
+int             crs.nCountersPerModuleW1         =   16;  //per layer
+int             crs.nCountersPerModuleW2         =   16;
+int             crs.nCountersPerModuleW3         =   16;
+int             crs.nCountersPerModuleW4         =   16;
+
+//vector<double>  crs.firstCounterW1               =  {-3904.0,     0.0, 20000.0};
+//vector<double>  crs.firstCounterW2               =  {-3904.0, -225.25, 20000.0};
+//vector<double>  crs.firstCounterW3               =  {-3904.0, -450.50, 20000.0};
+//vector<double>  crs.firstCounterW4               =  {-3904.0, -675.75, 20000.0};
+vector<double>  crs.firstCounterW1               =  {-3904.0, -1500.00, 39000.0};
+vector<double>  crs.firstCounterW2               =  {-3904.0, -1725.25, 39000.0};
+vector<double>  crs.firstCounterW3               =  {-3904.0, -1950.50, 39000.0};
+vector<double>  crs.firstCounterW4               =  {-3904.0, -2175.75, 39000.0};
+
+vector<double>  crs.layerDirectionW1             =  {0, 1, 0};
+vector<double>  crs.layerDirectionW2             =  {0, 1, 0};
+vector<double>  crs.layerDirectionW3             =  {0, 1, 0};
+vector<double>  crs.layerDirectionW4             =  {0, 1, 0};
+
+vector<double>  crs.offsetDirectionW1            =  {0, 0, 1};
+vector<double>  crs.offsetDirectionW2            =  {0, 0, 1};
+vector<double>  crs.offsetDirectionW3            =  {0, 0, 1};
+vector<double>  crs.offsetDirectionW4            =  {0, 0, 1};
+
+vector<double>  crs.gapDirectionW1               =  {0, 0, 1};
+vector<double>  crs.gapDirectionW2               =  {0, 0, 1};
+vector<double>  crs.gapDirectionW3               =  {0, 0, 1};
+vector<double>  crs.gapDirectionW4               =  {0, 0, 1};
+
+int crs.precedingSectorForW1  = -1;   //no preceding sector
+int crs.precedingSectorForW2  = -1;
+int crs.precedingSectorForW3  = -1;
+int crs.precedingSectorForW4  = -1;
+
+int crs.sectorTypeW1  = 1;
+int crs.sectorTypeW2  = 2;
+int crs.sectorTypeW3  = 3;
+int crs.sectorTypeW4  = 4;
+
+bool crs.sipmsAtSide0W1 = true;
+bool crs.sipmsAtSide0W2 = true;
+bool crs.sipmsAtSide0W3 = true;
+bool crs.sipmsAtSide0W4 = true;
+
+bool crs.sipmsAtSide1W1 = false;
+bool crs.sipmsAtSide1W2 = false;
+bool crs.sipmsAtSide1W3 = false;
+bool crs.sipmsAtSide1W4 = false;
+
+int crs.FEBBoxesAtSide0W1  = 0;
+int crs.FEBBoxesAtSide0W2  = 0;
+int crs.FEBBoxesAtSide0W3  = 0;
+int crs.FEBBoxesAtSide0W4  = 0;
+
+int crs.FEBBoxesAtSide1W1  = 0;
+int crs.FEBBoxesAtSide1W2  = 0;
+int crs.FEBBoxesAtSide1W3  = 0;
+int crs.FEBBoxesAtSide1W4  = 0;
+
+//bool g4.doSurfaceCheck    = true;
+int  g4.nSurfaceCheckPointsPercmsq =   1;
+int  g4.minSurfaceCheckPoints      = 100; // per volume
+int  g4.maxSurfaceCheckPoints = 10000000; // per volume
+
```

Comment:
I see that you have the long list of changing material to G4_AIR in ./Mu2eG4/geom/geom_common_trackerVST.txt .

Are the lists identical?  If so please make one file that has the content and #include it where needed.

If the lists are nearly identical, please put the common parts in one file and #include it.

Reply:
I didn't realize that geom_common_trackerVST.txt tried to do something similar (turn the concrete into air). I left the dirt unchanged in geom_Wideband4modules_v3.txt, while geom_common_trackerVST.txt also removed the dirt and probably other things as well.

Reply:
Yes, I can update the CMakeList.txt

Reply:
I removed the change in SConscript, because it is no longer needed. I only used it for a test.

Reply:
OK.